docs/whatsnew/0.2.rst

```diff
@@ -4,6 +4,5 @@
 What's New in Astropy 0.2
 *************************
 
-See this page in the `Astropy v0.2 documentation`__.
+``See this page in the `Astropy v0.2 documentation`__.``
```

Comment:
```suggestion
See this page in the ``Astropy v0.2 documentation``.
```

Reply:
I have addressed this in the most recent commit.